ocaml/xapi-storage/python/xapi/storage/api/volume.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import print_function
-from xapi import success, Rpc_light_failure, InternalError, UnmarshalException, TypeError, is_long, UnknownMethod
+from xapi import success, Rpc_light_failure, InternalError, UnmarshalException, TypeError, UnknownMethod, is_long, is_str
```

Comment:
Is this PR incomplete? is_str is unused, TypeError is suspect that needs to be imported

Reply:
`ocaml/xapi-storage/python/xapi/__init__.py` redefines it as a custom class here:

https://github.com/xapi-project/xen-api/blob/feature/py3/ocaml/xapi-storage/python/xapi/__init__.py#L127

There is so much wrong with this initially generated code that not everything can be fixed at once: Added sorting the imports and added disabling excessive `pylint` warnings.

We can fix those only piece by piece, step by step. The PR should be reviewable now.